migration/20170606-1-move-authentication-configuration-to-external-integrations.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python
```

Comment:
This filename doesn't need the `1-` on it. It will still run the same, but the counter isn't required.